.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 10.x]
+        sequelize-version: [null, next]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install sequelize@${{ matrix.sequelize-version }}
+      if: matrix.sequelize-version != null
+    - name: run ci
+      run: |
+        npm install
+        npm run lint
+        npm run test -- --coverage
+      env:
+        CI: true
+    - name: Coverage
+      uses: codecov/codecov-action@v1
+      if: matrix.node-version == '12.x' && matrix.sequelize-version == null
```

Comment:
Questions (I'm learning how to setup GitHub CI by seeing what you wrote):

* Why is there a `-` in some of the lines (12, 13, 17, 19, 26) but not all of them?
* I am confused by indentation, it looks like the `if:` lines are 'inside' the above `name`/`uses` specifications but they are not indented?

(These might be more yaml questions than GitHub CI questions)

Reply:
Yeah, yaml takes a bit of getting used to - it's one of those languages that's easier to write than read. The `-`s are for list items, so each `-` under `steps` is a single step, represented by an object with properties (`uses`, `run`, `with`). An easy way to tell what's going on is with a yaml-to-json converter like https://www.convertjson.com/yaml-to-json.htm - you can paste in [this file](https://raw.githubusercontent.com/sequelize/umzug/5b1bfff23059dbd43522ebceb6e9909f624be91d/.github/workflows/ci.yml)

GitHub actions docs are also pretty good at explaining what each section does.

Reply:
Nice!! Thanks for that link. I got it now.

P.S. you can merge the PR :)

Reply:
Forgot to actually merge, thanks for the reminder!